diff --color -ru zabbix-6.4.4/configure.ac zabbix-6.4.4_cgoflags-append-fix/configure.ac
--- zabbix-6.4.4/configure.ac	2023-06-27 09:15:06.000000000 +0000
+++ zabbix-6.4.4_cgoflags-append-fix/configure.ac	2023-07-15 06:54:02.153667094 +0000
@@ -1837,7 +1837,7 @@
 fi
 
 if test "x$agent2" = "xyes"; then
-	CGO_CFLAGS="$TLS_CFLAGS $LIBPCRE_CFLAGS $LIBPCRE2_CFLAGS"
+	CGO_CFLAGS="$CGO_CFLAGS $TLS_CFLAGS $LIBPCRE_CFLAGS $LIBPCRE2_CFLAGS"
 	CGO_LDFLAGS="$AGENT2_LDFLAGS $LD_START_GROUP \
 		\${abs_top_builddir}/src/libs/zbxalgo/libzbxalgo.a \
 		\${abs_top_builddir}/src/libs/zbxcommon/libzbxcommon.a \
